modules/ROOT/pages/deploy-to-cloudhub.adoc

```diff
@@ -99,9 +99,11 @@ If not set, by default this value is set to +https://anypoint.mulesoft.com+. | N
 | `muleVersion` | The Mule runtime engine version to run in your CloudHub instance.  +
 Ensure that this value is equal to or higher than the earliest required Mule version of your application. +
 Example value: `4.3.0` +
-
-Starting with Mule 4.5, deployments to CloudHub require Major.Minor version, which deploys the latest version of Mule runtime. 
-Example value: `4.5`| Yes
+Starting with Mule 4.5, deployments to CloudHub accepts the Mule version exact name or the semantic versioning, which deploy the latest version of Mule runtime. +
```

Comment:
```suggestion
Starting with Mule 4.5, deployments to CloudHub accept the Mule version exact name or the semantic versioning, which deploys the latest version of Mule runtime. +
```

Reply:
I assume the plus at the end of the line in a table is a line break?

Reply:
It is!